manifests/artemis.pp

```diff
@@ -8,40 +8,40 @@
     ensure  => file,
     content => template('candlepin/broker.xml.erb'),
     mode    => '0640',
-    owner   => 'tomcat',
-    group   => 'tomcat',
+    owner   => $candlepin::user,
```

Comment:
Like `concat{ $candlepin::candlepin_conf_file:`, all of these files can be owned by root I think. No need to change them at runtime by the application, right?

Almost starts to feel like that should be its own commit.